src/plugins/dashboard/public/dashboard_container/embeddable/dashboard_container.tsx

```diff
@@ -930,7 +930,7 @@ export class DashboardContainer
         return;
       }
 
-      panelRef.scrollIntoView({ block: 'center' });
+      return panelRef.scrollIntoView({ block: 'nearest' });
```

Comment:
Why return this?

Reply:
I had it in an if block before and forgot to remove the return - good catch, thanks!